model/build-processor/src/main/java/org/keycloak/models/map/processor/GenerateEntityImplementationsProcessor.java

```diff
@@ -99,6 +103,10 @@ protected void afterAnnotationProcessing() {
         }
     }
 
+    protected void generatedAnnotation(final PrintWriter pw) {
```

Comment:
Should we use this method also in [HotRod processor](https://github.com/keycloak/keycloak/blob/main/model/build-processor/src/main/java/org/keycloak/models/map/processor/GenerateHotRodEntityImplementationsProcessor.java)?